CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ If you make a pull request that's best reviewed by a specific person, tag or ass
 
 ### How to review and merge pull requests
 
-Any TTS staff member can review a pull request. If it's not your area of expertise, ask somebody who has expertise in that area (for instance, by commenting to tag a person or sharing the PR in Slack).
+Any TTS staff member can review a pull request. If it's not your area of expertise, ask somebody who has expertise in that area (for instance, by commenting to tag a person or sharing the PR in Slack). If the author of the pull request added a specific person as a reviewer, hold off merging the pull request until the specified reviewer has had a chance to review it. 
```

Comment:
a good bit of advice I frequently fail to follow and often regret.

What if they added multiple reviewers? Wait for one, or wait for all?